tests/variables.tf

```diff
@@ -0,0 +1,84 @@
+variable "serviceregistry_api" {
+  type        = string
+  description = "Service registry API endpoint to test against"
+
+  default = "https://serviceregistry-production-http.managed.services.opendoor.com"
```

Comment:
Remove this before publishing this repo?

Reply:
done